src/app/main/component/ubs/services/order.service.ts

```diff
@@ -82,7 +82,7 @@ export class OrderService {
     this.setOrder(order);
   }
 
-  getOrderUrl(): Observable<Order> {
+  getOrderUrl(): Observable<any> {
```

Comment:
fix this issue getOrderUrl(): Observable<Order | any>
because an error appears:  type 'string' not assignable to type 'string'
at my stage of study - it`s a bit hard to eliminate the problem.